index.html

```diff
@@ -88,7 +90,7 @@ <h1 id="filter">Recent Site</h1>
             <h2><span id="view">Graph</span> View</h2>
             <!-- @todo display list or graph view, update view based on filter and controls -->
             <!-- @todo remove the hard-coded width, height values -->
-            <canvas id="canvas" width="100%" height="100%"></canvas>
+            <svg width="700" height="700"></svg>

```

Comment:
The D3 library uses a property called `d3.forceCenter(width/2, height/2)` that centers the vis in the `<svg>` wrapper element. It does not accept percentage-based values and was defaulting to (0,0), so I reverted it to pixels again for the initial visualization to display properly.

I therefore need to compute the pixel height after the <svg> wrapper element for the vis has been rendered and pass that explicit (x,y) value in. Since I don't currently style this `<svg>` wrapper element in CSS, `element.clientWidth` and `element.clientHeight` return 0 (see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientWidth)), so I have used `element.getBoundingClientRect()`.